controllers/designate_controller.go

```diff
@@ -831,7 +831,7 @@ func (r *DesignateReconciler) reconcileNormal(ctx context.Context, instance *des
 		}
 		Log.Info(fmt.Sprintf("pools.yaml content is\n%v", poolsYaml))
 		updatedPoolsYaml := make(map[string]string)
-		updatedPoolsYaml[designate.PoolsYamlsConfigMap] = poolsYaml
+		updatedPoolsYaml["pools-yaml-content"] = poolsYaml
```

Comment:
I'm curious why the switch to from using a constant to a hard coded string here?